resources/definitions/Relay.graphql

```diff
@@ -144,3 +144,8 @@ directive @prependNode(
     edgeTypeName: String!
 ) on FIELD
 
+"""
+A directive added to mutations which tells Relay to generate types that cover the optimisticResponse parameter to commitMutation
+"""
+directive @raw_response_type on MUTATION
```

Comment:
Looks like it also can be applied to queries and subscriptions (https://github.com/facebook/relay/blob/9c14d82fc2f7fd14d24a8b276c5a0c9cd9ce3078/compiler/crates/relay-schema/src/relay-extensions.graphql#L30):

```graphql
"""
A directive added to queries which tells Relay to generate types that cover
the optimisticResponse parameter to commitMutation.
"""
directive @raw_response_type on QUERY | MUTATION | SUBSCRIPTION
```

Reply:
Interesting that they also allow this directive to be added to queries and mutations as they don't need this type... but I'll align the directive definitions to match the official ones👍🏼